src/components/location-search/location-search.tsx

```diff
@@ -0,0 +1,118 @@
+import debounce from 'lodash.debounce';
+import { ComboBox, ESystemIconNames } from '@mdb/flora';
+import { ChangeEvent, useCallback, useMemo, useState } from 'react';
+
+import { getURLPath } from '../../utils/format-url-path';
+
+type Option = string | { icon: string; label: string };
+
+// TODO: needs to be updated with icon names that will be added by Web Team - https://jira.mongodb.org/browse/WEBSITE-13740
+const DEFAULT_OPTIONS = [
+    { icon: ESystemIconNames.ARROW_UP, label: 'Current Location' },
+    { icon: ESystemIconNames.ARROW_DOWN, label: 'Virtual' },
+];
+
+interface LocationSearchProps {
+    onSelectCallback?: (selection: string) => void;
+}
+
+export default function LocationSearch({
```

Comment:
i think we may want to consider moving some of this logic, especially state/value stuff, to `useSearch` for the sake of consistency, since the other three search components' searching logic resides there currently. 

also, if i'm reading the code right, i don't believe there will be a way to clear the location search from the parent component, which would be necessary (i think? can't find a figma but that's the way it is on other content type pages) for the no results found "Back to Events" button on the events content type page. e.g. a line could be added in this function https://github.com/mongodb/devcenter/blob/68e1df61db432b891e0b4f97b21a9f043492ab63/src/hooks/search/index.ts#L252 to facilitate clearing

Reply:
yeah thats a fair point, i built out this component in isolation for now just to know the Google API's and the ComboBox were working. ill look into how this will tie in to the existing search functionality